.github/workflows/aws-scipool.yaml

```diff
@@ -58,5 +58,6 @@ jobs:
           mkdir -p templates/remote
           sceptre delete --yes prod/sc-product-assoc-ec2-linux-docker.yaml
           sceptre delete --yes prod/sc-product-assoc-ec2-linux-jumpcloud-notebook.yaml
+          sceptre delete --yes prod/sc-product-assoc-ec2-linux-docker-notebook.yaml
```

Comment:
you added the product for all production environments (prod, bmgfki, strides) so you will need an entry for each.

Also i would consider only enabling this product on prod for further testing then promote it to  bmgfki and strides so that it gets testing from internal sage folks before making available to more external users.   Just a thought but it's up to you.

Reply:
> you added the product for all production environments (prod, bmgfki, strides) so you will need an entry for each.

This is the change you made in #1071 , correct?

> Also i would consider only enabling this product on prod for further testing then promote it to bmgfki and strides so that it gets testing from internal sage folks before making available to more external users. Just a thought but it's up to you.

Once we've tested in 'develop' I'd like to promote to all 3 production environments.

Reply:
> This is the change you made in #1071 , correct?
no, that PR does not add the `linux-docker-notebook` product.

Reply:
> This is the change you made in #1071 , correct?

no, that PR does not add this product (linux-docker-notebook) to the list.